<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="button0">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>470</y>
     <width>187</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="Numberbutton" name="button1">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>176</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>176</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>176</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button5">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>273</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button6">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>273</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button7">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>371</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button8">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>371</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button9">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>371</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>273</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDot">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>470</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEqual">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPlus">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>176</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>53</width>
     <height>59</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>➕</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonMinus">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>273</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>➖</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonMultiply">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>✖️</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDivide">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>79</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>➗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAC">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>79</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBackspace">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>79</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>⬅️</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPercent">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>79</y>
     <width>92</width>
     <height>104</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>374</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>46</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <zorder>buttonPlus</zorder>
  <zorder>button0</zorder>
  <zorder>button1</zorder>
  <zorder>button2</zorder>
  <zorder>button3</zorder>
  <zorder>button5</zorder>
  <zorder>button6</zorder>
  <zorder>button7</zorder>
  <zorder>button8</zorder>
  <zorder>button9</zorder>
  <zorder>button4</zorder>
  <zorder>buttonDot</zorder>
  <zorder>buttonEqual</zorder>
  <zorder>buttonMinus</zorder>
  <zorder>buttonMultiply</zorder>
  <zorder>buttonDivide</zorder>
  <zorder>buttonAC</zorder>
  <zorder>buttonBackspace</zorder>
  <zorder>buttonPercent</zorder>
  <zorder>label</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Numberbutton</class>
   <extends>QPushButton</extends>
   <header location="global">Numberbutton.h</header>
   <slots>
    <signal>signal1()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
